src/baconian.rs

```diff
@@ -0,0 +1,467 @@
+//! Bacon's cipher or the Baconian cipher is a method of steganography
+//! (a method of hiding a secret message as opposed to just a cipher) devised by Francis Bacon in 1605.
+//! A message is concealed in the presentation of text, rather than its content.
+//!
+//! Each character of the message plaintext is encoded as a 5-bit binary,
+//!  these are then "hidden" in a decoy message through the use of font variation.
+//!
+//! This cipher is very easy to crack, once the method of hiding is known, therefore this
```

Comment:
I would suggest changing this to:

"
Each character of the plaintext message is encoded as a 5-bit binary character. These characters are then "hidden" in a decoy message through the use of font variation. This cipher is very easy to crack once the method of hiding is known. As such, this implementation includes the option to set whether the substitution is distinct for the whole alphabet, or whether it follows the classical method of treating 'I' and 'J', and 'U' and 'V' as interchangeable characters (as would have been the case in Bacon's time). 

Additionally, this implementation allows the user to change the underlying binary character choice. This is traditionally 'a' and 'b', but the user can choose any pair of characters. If no concealing text is given and the boilerplate of "Lorem ipsum..." is used, a plaintext message of up to 50 characters may be hidden.
"

Reply:
Done. (with amendments...)